VotingApplication/VotingApplication.Web/Scripts/Controllers/PointsVoteController.js

```diff
@@ -1,86 +1,87 @@
 ﻿(function () {
-    angular.module('GVA.Voting').controller('PointsVoteController', ['$scope', 'IdentityService', 'PollService', 'TokenService', function ($scope, IdentityService, PollService, TokenService) {
+    angular.module('GVA.Voting').controller('PointsVoteController', ['$scope', '$routeParams', 'IdentityService', 'PollService', 'TokenService',
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.